### PR TITLE
Proxy support and refactor codes

### PR DESCRIPTION
**Pull request by eggplants**
Proxy feature will be worked if `ALL_PROXY` or `HTTPS_PROXY` is set.

---

@narugo1992 Please check.

```bash
pip install git+https://github.com/eggplants/get-pixivpy-token@proxy-support

export ALL_PROXY="..."
gppt l
gppt lh -u "..." -p "..."
```

### CODE REVIEW

**Comment by narugo1992** [comment]
I see, now the `gppt l` is okay, but something wrong with `gppt lh`.

File: gppt/_selenium.py [hunk 1]
```diff
@@ -176,22 +174,35 @@ def __try_login(self) -> None:
...
+
+        if headless:
+            options.add_argument("--headless")
+            options.add_argument("--disable-gpu")
+            options.add_argument("--disable-extensions")
+            options.add_argument("--disable-infobars")
+            options.add_argument("--disable-dev-shm-usage")
+            options.add_argument("--disable-browser-side-navigation")
+            options.add_argument('--proxy-server="direct://"')
+            options.add_argument("--proxy-bypass-list=*")
```
Comment (narugo1992) [inline gppt/_selenium.py:188]: These lines should be removed. Now when you try to login pixiv with headless mode, the `--proxy-server` and `--proxy-bypass` arguments will be added twice.

```python
options.add_argument('--proxy-server="direct://"')
options.add_argument("--proxy-bypass-list=*")
```

Comment (narugo1992) [reply gppt/_selenium.py:188]: After remove these lines, now `gppt lh` is okay to me 😄

Comment (eggplants) [reply gppt/_selenium.py]: Thanks. Fixed: 279783258327d1bc36107d9022df11d14003edb4

**Comment by eggplants** [comment]
@narugo1992 Thank you for your contributions!

**Pull request merged**